lib/datadog/appsec/event.rb

```diff
@@ -137,6 +137,24 @@ def build_service_entry_tags(event_group)
         end
         # rubocop:enable Metrics/MethodLength
 
+        def add_event_tags(scope, waf_result)
```

Comment:
Since the class is already named `Event`, should we rename this function to `add_tags`?

Reply:
Fixed!